src/utils/format.ts

```diff
@@ -158,3 +159,22 @@ export const formatNationalitiesToSentenceCase = (nationality: string | undefine
   });
 };
 
+export const formatAddress = (addressFields: (string | undefined)[]): string => {
+  return addressFields
+    .filter(field => field != undefined && field !== "")
```

Comment:
.map(s => s?.trim())
.filter(field => field !== undefined && field !== "")
.join(',  ')

Note !== instead of !=

Reply:
Fixed